packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -54,6 +57,73 @@ List<String> debugDescribeTransform(Matrix4 transform) {
   return matrix;
 }
 
+bool _extensionsInitialized = false;
+
+void initServiceExtensions() {
+  if (_extensionsInitialized)
+    return;
+
+  registerExtension('flutter', _flutter);
+  // TODO: Expose debugDumpApp().
+  // TODO: Expose StatisticsOverlay.
+  registerExtension('flutter.debugPaint', _debugPaint);
+  registerExtension('flutter.timeDilation', _timeDilation);
+
+  _extensionsInitialized = true;
+
+  // Emit an info level log message; this tells the debugger that the Flutter
+  // service extensions are registered.
+  log('Flutter initialized', name: 'flutter', level: 800);
+}
+
+/// Just respond to the request. Clients can use the existence of this call to
+/// know that the debug client is a Flutter app.
+Future<ServiceExtensionResponse> _flutter(String method, Map parameters) {
```

Comment:
Give type parameters to the Map. (Flutter's style guide requests that you avoid any implicit "dynamic"s.)